docs/user/dashboard/lens.asciidoc

```diff
@@ -402,3 +402,19 @@ Pagination in a data table is unsupported. To use pagination in data tables, cre
 ====
 Specifying the color for a single data point, such as a single bar or line, is unsupported.
 ====
+
+[discrete]
+[[is-it-possible-to-inspect-the-elasticsearch-queries-in-Lens]]
+.*Is it possible to inspect the Elasticsearch queries in Lens?*
+[%collapsible]
+====
+You can inspect the requests sent by the visualization to Elasticsearch by using the Inspector. It can be accessed within the editor or in the dashboard.
```

Comment:
```suggestion
To view the requests that visualizations send to {es}, click [add user action for Lens].
```

Reply:
It is not clear to me if `[add user action for Lens]` is a placeholder or an actual link to a page 😅

Reply:
Sorry! It's a placeholder.